config/locales/fr.yml

```diff
@@ -38,17 +39,31 @@ fr:
     people_who_follow: Personnes qui suivent %{name}
     posts: Statuts
     remote_follow: Suivre à distance
+    reserved_username: Ce nom d'utilisateur est réservé
```

Comment:
You should use « Nom d'utilisateur⋅ice » in order to stay consistent with the other translated strings. 🙂